extensions/Cake.GithubUtility.yml

```diff
@@ -0,0 +1,16 @@
+Type: Addin
+Name: Cake.GithubUtility
+NuGet: Cake.GithubUtility
+Assemblies:
+- /**/Cake.GitHubUtility.dll
+Repository: 
```

Comment:
Repo: https://github.com/wk-j/cake-github-utility

Reply:
Manually updating the repository URL in the YAML file is not a practical option because it gets overwritten everytime the discoverer analyses an addin. Better solution would be to raise an issue in the addin repo to encourage the author to update their nuspec.